### ISSUE TITLE
About finetuning on UCF101

### ISSUE THREAD

**Issue by xxbbml** [qijiezhao/pseudo-3d-pytorch#7]
Hi~
Thanks for sharing the codes and some experiments results.
Recently I have also tried finetuing the P3D199 Kinetics models on UCF101, but only got about 85% accuracy of action recognition task. I just wonder do you have any other tricks or did I make some mistakes..? Thanks!

**Comment by qijiezhao** [comment]
Yes,  training tricks are deeply required if you want to get a high score. I could share you some processes like jittering, TSN and corner-crop testing,  every step is necessary.

**Comment by xxbbml** [comment]
Thanks. I'll try some data augmentation methods.
And could you share a little more about how to use TSN on this model?  Thanks!

**Comment by qijiezhao** [comment]
Mainly modify the data input method so that you could select multiple segments together to train for each sample. More details you may take a look at xiong's official TSN repo: https://github.com/yjxiong/tsn-pytorch/blob/master/dataset.py. His code also supports 10-crop testing. 
FYI

**Comment by icoz69** [comment]
hi, did you try any tricks to improve the results?  i got similar accuracy
